src/main/java/org/team1540/elmo/subsystems/drivetrain/EmergencyBrake.java

```diff
@@ -0,0 +1,21 @@
+package org.team1540.elmo.subsystems.drivetrain;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class EmergencyBrake extends CommandBase {
```

Comment:
I added this because we had a lot of buttons left over and Dale suggested to add one after I asked on what I should add there. It doesn't carry that much use, but it might have a pratical use.

Reply:
I'm going to recommit the files afterwards